doctest/bootstrap.sh

```diff
@@ -14,9 +14,13 @@ fi
 
 $PYTHON -m venv $DIR/.venv
 if [ ! -f $DIR/.venv/bin/pip ]; then
-    wget https://bootstrap.pypa.io/get-pip.py
-    $DIR/.venv/bin/python get-pip.py
-    rm -f get-pip.py
+    if [ "$(expr substr $(uname -s) 1 10)" == "MINGW32_NT" ] || ["$(expr substr $(uname -s) 1 10)" == "MINGW64_NT" ]; then
```

Comment:
AFAIK windows default shell is `PowerShell`, so you need another script and if condition in github workflow. Alternative option - add a script call from `build.gradle` for `:doctest` on Windows only.

Reply:
Thanks for the tip - will check it out